static/app/views/issueList/actions/index.tsx

```diff
@@ -136,7 +138,11 @@ function ActionsBarPriority({
           )}
           {!anySelected && (
             <HeaderButtonsWrapper key="sort" {...animationProps}>
-              <IssueListSortOptions sort={sort} query={query} onSelect={onSortChange} />
+              {!organization.features.includes('issue-stream-table-layout') ? (
+                <IssueListSortOptions sort={sort} query={query} onSelect={onSortChange} />
+              ) : (
+                <ToolbarHeader>{t('Issue')}</ToolbarHeader>
+              )}
```

Comment:
Could be argued that replacing the dropdown with the "issue" text is out of the scope of this PR, happy to remove this change if anyone feels strongly about it.